src/app/rosetta/lib/account.ml

```diff
@@ -30,11 +30,6 @@ end
 
 module Sql = struct
   module Balance_from_last_relevant_command = struct
-    let max_txns =
```

Comment:
Was this just ... unused?

Reply:
Looks exactly this way. We shouldn't turn warnings off less freely, it seems so.